generate.sh

```diff
@@ -5,11 +5,12 @@ set -o pipefail
 usage()
 {
   echo "Common settings:"
-  echo "  --platform <value>         Platform: MacOSX, iPhoneOS, iPhoneSimulator, AppleTVOS, AppleTVSimulator"
+  echo "  --platform <value>         Platform: iPhoneOS, iPhoneSimulator, AppleTVOS, AppleTVSimulator, MacOSX"
   echo "  --version <value>          Version of the SDK"
-  echo "  --arch <value>             Architecture: arm64e-apple-macos, x86_64-apple-macos"
   echo "  --framework <value>        Framework to generate bindings for"
   echo "  --configuration <value>    Configuration: Debug, Release"
+  echo "  --maccatalyst              Use Mac Catalyst instead of iOS"
```

Comment:
Why do we need special argument for MacCatalyst if we have the `--platform` argument already?

I think it might be better to specify it directly as an option if we need to to distinguish between `MacOSX` and `MacCatalyst` rather than using `iPhoneOS` + `--maccatalyst`.

Reply:
Initially, I wrapped it with iPhoneOS as the target is `apple-ios-macabi`. Now it is a different platform.